challenges/stats_preview_card_component/tailwind.config.js

```diff
@@ -28,7 +28,7 @@ module.exports = {
 
       screens: {
         mobile: '375px',
-        desktop: '1440px',
+        desktop: '1200px',
       },
```

Comment:
Use default Tailwind breakpoints.

Reply:
Changed to default Tailwind breakpoints.